peps/converters.py

```diff
@@ -10,11 +10,21 @@
 from django.core.files import File
 from django.db.models import Max
 
+from pygments import highlight
+from pygments.lexers import PythonLexer
+from pygments.formatters import HtmlFormatter
+
 from pages.models import Page, Image
 
 PEP_TEMPLATE = 'pages/pep-page.html'
 pep_url = lambda num: 'dev/peps/pep-{}/'.format(num)
 
+# To simplify syntax highlighting, all literal blocks (those produced by ::)
+# in the following PEPs will be automatically highlighted using Python lexer.
+# PEP editors/authors could make simple PRs extending this list.
+# This will be not needed when PEPs are moved to RtD and all code blocks are
+# formatted using .. code:: language.
+PURE_PYTHON_PEPS = [483, 484, 526]
```

Comment:
Seems like it would make more sense to update the PEPs with appropriate code blocks rather than do this. I think this would also remove the need for the following lines (although I'm not sure what "Fix highlighting code" is currently doing.

Reply:
I agree, I'll comment below what's happening.

Reply:
This is no longer used.